File: CodeForces/401-500/405/A/405A.php
<?php

// Problem 405A: Gravity Flip
// https://codeforces.com/contest/405/problem/A
// Executed with: PHP 8.2.29 (cli) (built: Jul  3 2025 16:16:05) (NTS)
// Linux version: Linux kernel 6.1.0-10-amd64
// Written on: October 23rd, 2025
// Codeforces language used: Cannot be submitted

// This isn't relevant here, but we have to take it in anyway
$w = intval(trim(fgets(STDIN)));

$line = trim(fgets(STDIN));
$arr = array_map('intval', explode(' ', $line));

sort($arr);

for ($x = 0; $x < count($arr); $x++) {
    echo $arr[$x];
    echo " ";
}
echo "\n";
?>

File: CodeForces/401-500/405/A/405A.go
// Problem 405A: Gravity Flip
// https://codeforces.com/contest/405/problem/A
// Executed with: go version go1.24.4 linux/arm64
// Linux version: Linux kernel 6.17.0-5-generic
// Submitted on: October 26th, 2025
// Codeforces language used: Go 1.22.2

package main

import (
	"fmt"
	"slices"
)

func main() {
	var nums int
	fmt.Scan(&nums)
	var array []int
	for i := 0; i < nums; i++ {
		var temp int
		fmt.Scan(&temp)
		array = append(array, temp)
	}
	slices.Sort(array)
	for i := 0; i < nums-1; i++ {
		fmt.Print(array[i])
		fmt.Print(" ")
	}
	fmt.Println(array[nums-1])
}
